updater/spec/dependabot/updater/operations/group_update_all_versions_spec.rb

```diff
@@ -326,6 +326,29 @@
     end
   end
 
+  context "when there are semver rules but an error occurs gathering versions" do
+    before do
+      allow_any_instance_of(Dependabot::Bundler::UpdateChecker).
+        to receive(:latest_version).
+          and_raise(StandardError.new("Test error while getting latest version"))
```

Comment:
```suggestion
        and_raise(StandardError.new("Test error while getting latest version"))
```

The linter is upset about this